signin.js

```diff
@@ -0,0 +1,116 @@
+// ID를 통해 DOM 요소에 접근
+const signinForm = document.querySelector("signinForm");
+
+// 이메일
+const email = document.querySelector('#email');
+const emailInput = document.querySelector('.emailEmptyError');
+const emailFalse = document.querySelector('.emailFalseError');
+
+// 비밀번호
+const password = document.querySelector('#password');
+const passwordInput = document.querySelector('.passwordEmptyError');
+const passwordFalse = document.querySelector('.passwordFalseError');
+
+// 버튼
+const submitBtn = document.querySelector('.sign_div form button[type="submit"]');
+
+// 오류 메시지
+function showError(input, errorId) {
+    const errorElement = document.getElementById(errorId);
+    errorElement.style.display = "block";
+    input.style.border = "2px solid #f74747";
+}
+
+// 상태 초기화
+function hideError(input, errorId) {
+    const errorElement = document.getElementById(errorId);
+    errorElement.style.display = "none";
+    input.style.border = "none";
+}
+
+// 이메일 유효성 검사
+function validateEmailString(email) {
+    const emailRegex = /^[A-Za-z0-9._%-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4}$/;
+    return emailRegex.test(email);
+};
+
+// 비밀번호 유효성 검사
+function validatePasswordString(password) {
+    return passwordRegex.length(password) >= 8;
```

Comment:
선언된 passwordRegex 변수가 없는 것 같습니다!

Reply:
그리고 단순히 길이로 검사하시는 거라면 이렇게 쓰셔야 할 것 같습니다.
```suggestion
    return password.length >= 8;
```

Reply:
length는 함수 형태의 메서드가 아닌 길이 값 자체를 반환해 줍니다.
[공식문서](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/String/length) 첨부드려요!